src/components/pages/Publications/PublicationsConstants.ts

```diff
@@ -353,6 +346,28 @@ const listOfMediaPublicationsProps: PublicationProps[] = [
         authors: "",
         url: "https://www.bloomberg.com/news/articles/2020-05-06/understanding-the-virus-and-its-unanswered-questions-quicktake",
         img: bloomberg
+    },
+    {
+        day: "12",
+        month: "January",
+        year: "2022",
+        titleKey1: 'IndustryMediaTitles',
+        titleKey2: ['EconomistDeathToll2'],
+        publicationName: "The Economist",
```

Comment:
Are we using this `publicationnName` field? If so, shoudn't it be translated?

Reply:
Since it is the name of the actual publication, I think we would just use The Economist (I do not know if they have a French website version). The field existed from before I came on and it was not translated then. I could add it in but I am not sure if we are supposed to or what the course of action in these situations is

Reply:
Ok fine for now, let's just merge